wizeline/remix-project-lab
# Fix/282/do not allow an end date to be selected that is less than the start date

# What does this PR do?
1. Establishes that an end date less than the start date cannot be selected on the form 2.
2. Makes the field of universities in the forms mandatory.
3. Instead of displaying the applicant's profile with his or her university, it shows his or her email address in /projects.



# Where should the reviewer start?
app/routes/applicationForm/$applicantId/index.tsx
app/core/components/LabeledTextField/index.tsx
app/core/components/ApplicantsSection.tsx



# How should this be manually tested?
In the applicant form, check that you are not allowed to select an end date less than the start date.



# Any background context you want to provide?



# What are the relevant tickets?
#282 



# Screenshots
<img width="565" alt="Captura de pantalla 2023-12-06 a la(s) 12 35 56 p m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/58eedbb3-1e44-4a0a-ac9c-e555f3d5bb64">



# Questions



# Checklist
